PROTO_tests/tests/io_test.py

```diff
@@ -140,17 +141,32 @@ def make_multi_dtype_dict():
     }
 
 
-class TestParquet:
-    par_test_base_tmp = f"{os.getcwd()}/par_io_test"
+@pytest.fixture
+def par_test_base_tmp(request):
+    par_test_base_tmp = "{}/.df_test".format(os.getcwd())
     io_util.get_directory(par_test_base_tmp)
```

Comment:
So this doesn't really matter since it's a tmp dir... but it looks like previously `par_test_base_tmp` ended with `"par_io_test"` and now it ends with `".df_test"`, which I figure wasn't intended since it makes less sense

Reply:
Fixed.